lib/Itho/IthoCC1101.cpp

```diff
@@ -324,21 +324,32 @@ bool IthoCC1101::parseMessageCommand() {
-  bool isTimer3Command   = checkIthoCommand(&inIthoPacket, ithoMessageTimer3CommandBytes);
-  bool isJoinCommand     = checkIthoCommand(&inIthoPacket, ithoMessageJoinCommandBytes);
-  bool isJoin2Command    = checkIthoCommand(&inIthoPacket, ithoMessageJoin2CommandBytes);
-  bool isRVJoinCommand   = checkIthoCommand(&inIthoPacket, ithoMessageRVJoinCommandBytes);
-  bool isLeaveCommand    = checkIthoCommand(&inIthoPacket, ithoMessageLeaveCommandBytes);
+  const bool isHighCommand     = checkIthoCommand(&inIthoPacket, ithoMessageHighCommandBytes);
+  const bool isRVHighCommand   = checkIthoCommand(&inIthoPacket, ithoMessageRVHighCommandBytes);
+  const bool isMediumCommand   = checkIthoCommand(&inIthoPacket, ithoMessageMediumCommandBytes);
+  const bool isRVMediumCommand = checkIthoCommand(&inIthoPacket, ithoMessageRVMediumCommandBytes);
+  const bool isLowCommand      = checkIthoCommand(&inIthoPacket, ithoMessageLowCommandBytes);
+  const bool isRVLowCommand    = checkIthoCommand(&inIthoPacket, ithoMessageRVLowCommandBytes);
+  const bool isRVAutoCommand   = checkIthoCommand(&inIthoPacket, ithoMessageRVAutoCommandBytes);
+  const bool isStandByCommand  = checkIthoCommand(&inIthoPacket, ithoMessageStandByCommandBytes);
+  const bool isTimer1Command   = checkIthoCommand(&inIthoPacket, ithoMessageTimer1CommandBytes);
+  const bool isTimer2Command   = checkIthoCommand(&inIthoPacket, ithoMessageTimer2CommandBytes);
+  const bool isTimer3Command   = checkIthoCommand(&inIthoPacket, ithoMessageTimer3CommandBytes);
+  const bool isJoinCommand     = checkIthoCommand(&inIthoPacket, ithoMessageJoinCommandBytes);
+  const bool isJoin2Command    = checkIthoCommand(&inIthoPacket, ithoMessageJoin2CommandBytes);
+  const bool isRVJoinCommand   = checkIthoCommand(&inIthoPacket, ithoMessageRVJoinCommandBytes);
+  const bool isLeaveCommand    = checkIthoCommand(&inIthoPacket, ithoMessageLeaveCommandBytes);
+
+  const bool isOrconStandByCommand = checkIthoCommand(&inIthoPacket, orconMessageStandByCommandBytes);
+  const bool isOrconLowCommand     = checkIthoCommand(&inIthoPacket, orconMessageLowCommandBytes);
+  const bool isOrconMediumCommand  = checkIthoCommand(&inIthoPacket, orconMessageMediumCommandBytes);
+  const bool isOrconFullCommand    = checkIthoCommand(&inIthoPacket, orconMessageFullCommandBytes);
+  const bool isOrconAutoCommand    = checkIthoCommand(&inIthoPacket, orconMessageAutoCommandBytes);
+  const bool isOrconTimer0Command  = checkIthoCommand(&inIthoPacket, orconMessageTimer0CommandBytes);
+  const bool isOrconTimer1Command  = checkIthoCommand(&inIthoPacket, orconMessageTimer1CommandBytes);
+  const bool isOrconTimer2Command  = checkIthoCommand(&inIthoPacket, orconMessageTimer2CommandBytes);
+  const bool isOrconTimer3Command  = checkIthoCommand(&inIthoPacket, orconMessageTimer3CommandBytes);
+  const bool isOrconAutoCO2Command = checkIthoCommand(&inIthoPacket, orconMessageAutoCO2CommandBytes);
```

Comment:
Is it really needed to check all possible types?
It looks like you may be done at some point as I think you will only have one of these set to true, right?

Reply:
That part already bothered me when I looked at it, but it's 'inherited' code.

I'll do some improvements there.